src/pages/Meeting/DetailMeatingManagePage.tsx

```
@@ -0,0 +1,200 @@
+                  <p className="">{groupName}</p>
+                { groups.length-1 == idx &&
+                  <button onClick={() => removeGroup(groupName)} className="text-red-500 hover:underline">삭제</button>
+                }
+                </div>
+                <hr className="border-t border-[#EAE5E2] mb-2" />
+                <div className="grid grid-cols-1 gap-y-2">
+                  {getMembersInGroup(groupName).map((member : ClubMember) => (
+                    <div
+                      className="bg-[#F4F2F1] w-[365px] h-[44px] rounded-[20px] flex items-center px-3 text-[17px] font-medium text-[#2C2C2C] truncate"
+                    >
+                      <img src={member.basicInfo.profileImageUrl} className="w-[32px] h-[32px] rounded-full bg-[#DADADA] mr-5 text-[var(--Gray-1,#2C2C2C)] font-pretendard text-[18px] font-medium leading-[135%]" />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)